package.json

```diff
@@ -1,25 +1,27 @@
 {
   "name": "cuttle",
   "description": "The deepest card game under the sea",
-  "version": "4.6.4",
+  "version": "4.7.0",
   "dependencies": {
-    "connect-redis": "3.0.2",
-    "dayjs": "^1.11.9",
-    "lodash": "^4.17.21",
-    "machinepack-passwords": "2.3.0",
-    "marked": "^5.1.1",
     "sails": "1.5.3",
+    "machinepack-passwords": "2.3.0",
     "sails-disk": "^2.1.2",
     "sails-hook-orm": "2.1.1",
     "sails-hook-sockets": "1.5.5",
     "sails-postgresql": "4.0.0",
     "sails.io.js": "1.2.1",
-    "socket.io-client": "2.3.0",
+    "connect-redis": "3.0.2",
+    "dayjs": "^1.11.9",
+    "lodash": "^4.17.21",
     "vue": "^3.3.4",
-    "vue-i18n": "^9.2.2",
+    "vuex": "^4.1.0",
     "vue-router": "^4.2.4",
     "vuetify": "^3.3.11",
-    "vuex": "^4.1.0"
+    "vue-i18n": "^9.2.2",
+    "vue-chartjs": "^5.2.0",
+    "chart.js": "^4.3.3",
+    "socket.io-client": "2.3.0",
+    "marked": "^5.1.1"
```

Comment:
I reordered deps so that backend deps go first, then deps used in both the backend + frontend i.e. lodash, then frontend deps, putting all the vue stuff at the top of frontend. This doesn't really matter but I think it's a bit easier to scan now